packages/react-native-sdk/src/providers/StreamCall.tsx

```diff
@@ -137,3 +153,57 @@ const ClearPushWSSubscriptions = () => {
   }, []);
   return null;
 };
+
+const eventEmitter =
+  NativeModules?.StreamVideoReactNative &&
+  new NativeEventEmitter(NativeModules?.StreamVideoReactNative);
```

Comment:
```suggestion
const eventEmitter =
  NativeModules?.StreamVideoReactNative 
    ? new NativeEventEmitter(NativeModules?.StreamVideoReactNative)
    : undefined;
```

Let's be a bit more explicit.